src/styles/index.scss

```
@@ -1,3 +1,121 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid transparent;
+  border-radius: 4px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__content {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    align-self: baseline;
+    color: #616070;
+    margin: 4px 0 16px;
```

I recommend you to use one style of adding margins. For example to add a vertical indent between elements use margin-bottom. To add horizontal one, use margin-right

So move margin-bottom: 4px to card__title
```suggestion
    margin-bottom: 16px;
```